resources/mod_ssl.rb

```diff
@@ -1,6 +1,10 @@
 include Apache2::Cookbook::Helpers
 unified_mode true
 
+property :mod_ssl_pkg, String,
+         default: 'mod_ssl',
+         description: ''
```

Comment:
I think we should have some kind of description, even if it is just this:

```suggestion
         description: 'The name of the mod_ssl package'
```